src/Model/Event.php

```diff
@@ -86,4 +86,9 @@ public static function getName(
 
         return "{$moment->value}.{$name}";
     }
+
+    public function getUseCaseRequest(): mixed
+    {
+        return $this->parameters['useCaseRequest'];
```

Comment:
technically this should not be added since the service proxy is generic and open source, so it should not know about the `useCaseRequest`, but pragmatically it avoids potential errors on our side, so we can go for it.